l10n_pt_invoice_spms/models/account_move_line.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2025 Dixmit
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class AccountMoveLine(models.Model):
+
+    _inherit = "account.move.line"
+
+    spms_prescription = fields.Char()
+    spms_user_number = fields.Char()
```

Comment:
```suggestion
    spms_user_number = fields.Integer()
```
@etobella This field is a number in both Oxigesti SPMS system and the SPMS documentation. Could you please update it accordingly?
<img width="911" height="558" alt="image" src="https://github.com/user-attachments/assets/045fc2cd-abea-4469-b3de-bfdd9d4e2a84" />

I did not find the official XSD, probably it's in PTplus module